README.md

```diff
@@ -78,6 +74,22 @@ cp -rf ~/ttcopy/bin ~/ttcopy/lib /usr/local
 echo "export PATH=$PATH:/usr/local/bin" >> ~/.zshrc" # if you need
 ```
 
+# Configuration
+First of all, `ttcopy` command displays the screen to let you set default ID and Password.
+Please prepare ID and Password **as you like**. Be lazy! You are **NOT** required to register them on any services.
+The data you copied can be pasted within the hosts having same ID and Password.
+
+```sh
+$ ttcopy
+Set default ID/Password.
+Enter ID for ttcopy/ttpaste:myid001   #<= Enter your ID ("myid001" is just an example).
+Enter password for ttcopy/ttpaste:    #<= Enter your password.
+Enter same password again:            #<= Enter again.
+
+Created credential file '/Users/user/.ttcopy/config'
```

Comment:
How about using `~/.config/ttcopy/config` -- some apps including git is using this config path. (cf. https://www.google.co.jp/search?q=%22~%2F.config%22 ) Or maybe we can use just `~/.ttcopy` if we are not going to create other config files.